verifiers/internal/gha/provenance.go

```diff
@@ -292,6 +292,12 @@ func isValidDelegatorBuilderID(prov iface.Provenance) error {
 	if len(parts) != 2 {
 		return fmt.Errorf("%w: %s", serrors.ErrorInvalidBuilderID, id)
 	}
+
+	// Exception for JReleaser builders.
+	// See https://github.com/slsa-framework/slsa-github-generator/issues/2035#issuecomment-1579963802.
+	if strings.HasPrefix(parts[0], JReleaserRepository) {
```

Comment:
I'm not sure how I feel about having one-off exceptions in verification. Especially since IIUC this is specific to JReleaser in that the single JReleaser action can work on a Java or Rust repo and we aren't especially likely to see this use case on other builders.

Reply:
I could not think of another way to handle it. Let me know if you have other ideas.